include/libdbc/message.hpp

```diff
@@ -29,8 +29,9 @@ struct Message {
 	ParseSignalsStatus parseSignals(const std::vector<uint8_t>& data, std::vector<double>& values) const;
 
 	void appendSignal(const Signal& signal);
-	const std::vector<Signal> signals() const;
+	const std::vector<Signal>& getSignals() const;

```

Comment:
I don't like the reference here since that ties it to the lifetime of the object now.

Reply:
Good point, I can change this

Reply:
@LinuxDevon I updated this